<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ClickerButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>TestNetwork</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ScoreLayout">
     <item>
      <widget class="QLabel" name="ScoreLabel">
       <property name="text">
        <string>Score:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ScoreValue">
       <property name="text">
        <string>0000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="TickLayout">
     <item>
      <widget class="QLabel" name="TickLabel">
       <property name="text">
        <string>Tick :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TickValue">
       <property name="text">
        <string>99999999999</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
